Dispatcher/UI/Profile/ProfileViewController.swift

```diff
@@ -8,8 +8,79 @@
 import UIKit
 
 class ProfileViewController: UIViewController {
-
+   
+    @IBOutlet var contentView: UIView!
+    @IBOutlet weak var profileHeaderView: UIView!
+    @IBOutlet weak var optionsTableView: UITableView!
+    
+    let profileViewModel = ProfileViewModel()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        styleHeaderShadow()
+        optionsTableView.dataSource = self
+        optionsTableView.delegate = self
+    }
+    
+    private func styleHeaderShadow() {
+        profileHeaderView.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.09).cgColor
+           profileHeaderView.layer.shadowOpacity = 0.6
+           profileHeaderView.layer.shadowOffset = CGSize(width: 0, height: 2)
+           profileHeaderView.layer.shadowRadius = 4
+           profileHeaderView.layer.masksToBounds = false
+    }
+    
+    private func handleLogout() {
+        self.profileViewModel.logout { errorMsg in
+            if let error = errorMsg {
+                self.present(createErrorAlert(error), animated: true, completion: nil)
+            } else {
+                self.performSegue(withIdentifier: SegueIdentifiers.fromLogoutToAuth, sender: self)
```

Comment:
Isn't the current screen being kept in the stack of screens?
Some iPhones have a swipe-to-go-back feature and in this case the user logged out but can still go back to the profile screen which is wrong